### ISSUE TITLE
Conflict with EDD Software Licensing

### ISSUE THREAD

**Issue by nosegraze** [wpninjas/edd-git-download-updater#35]
The JavaScript for this plugin hides the "Add New Upgrade Path" button in the Software Licensing extension. Due to this JavaScript:

    if ( 1 == this.useGit ) {
        $( '.edd_add_repeatable' ).hide();
        $( '.edd_remove_repeatable' ).hide();
        this.setUpgradeFile();
    }

I think this is for hiding the "Add New File" button? I just adjusted it to this on my own site to get it fixed:

    if ( 1 == this.useGit ) {
        $( '#edd_download_files .edd_add_repeatable' ).hide();
        $( '#edd_download_files .edd_remove_repeatable' ).hide();
        this.setUpgradeFile();
    }

**Comment by cklosowski** [comment]
I'm not seeing this behavior on my end, but those classes are pretty generically used inside the EDD Ecosystem, so making the hiding a little more strict is not a bad thing. Thanks @nosegraze!

**Comment by cklosowski** [comment]
@nosegraze Looks like this should't be happening. I'm seeing this in the current release `1.0.2`
```
				if ( 1 == this.useGit ) {
					$( '#edd_product_files .edd_add_repeatable' ).hide();
					$( '#edd_product_files .edd_remove_repeatable' ).hide();
					this.setUpgradeFile();
				}
```

Can you confirm this? If so I'll close this out.

**Comment by cklosowski** [comment]
This commit specifically should have fixed it: https://github.com/wpninjas/edd-git-download-updater/commit/00e1b94dd917cd39b4fecc30cde6ae8d4d4c2be1

**Comment by nosegraze** [comment]
Huh - I have my license key activated but never got the 1.0.2 update. I'm still on 1.0.1.

**Comment by cklosowski** [comment]
Ok, this was fixed in `1.0.2`, can you try and manually update using the .zip file available here?
https://github.com/wpninjas/edd-git-download-updater/releases/tag/1.0.2

You'll have to rename the extracted folder from `edd-git-download-updater-1.0.2` to `edd-git-download-updater` and upload it to your server. But I believe that should fix it for you.

**Comment by nosegraze** [comment]
Will do - thankyou.
